test/functional/wallet_getxpub.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test the getxpub RPC command."""
+
+from test_framework.authproxy import JSONRPCException
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import (
+    assert_equal,
+    assert_raises_rpc_error,
+)
+from test_framework.descriptors import descsum_create
+
+class WalletGetxpubTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.num_nodes = 2
+
+    def skip_test_if_missing_module(self):
+        self.skip_if_no_wallet()
+        self.skip_if_no_sqlite()
+
+    def run_test(self):
+        self.nodes[0].createwallet(wallet_name="w1")
+        w1 = self.nodes[0].get_wallet_rpc("w1")
```

Comment:
With a wallet different from the default one, I receive the error when sending to w2 address.
`test_framework.authproxy.JSONRPCException: Insufficient funds (-6)`

With the change below, it worked.

```suggestion
        w1 = self.nodes[0].get_wallet_rpc(self.default_wallet_name)
```